xbmc/rendering/dx/DeviceResources.h

```diff
@@ -81,6 +81,14 @@ namespace DX
 
     bool SetFullScreen(bool fullscreen, RESOLUTION_INFO& res);
 
+    void DetectDisplayHdrCapable(bool& hdr_capable, bool& hdr_enabled);
+    void SetHdrMetaData(DXGI_HDR_METADATA_HDR10& hdr10) const;
+    void ClearHdrMetaData() const;
+    bool Is10BitSwapchain() const { return m_Is10bSwapchain; }
+    void Set10BitSwapchain(bool flag) { m_Is10bSwapchain = flag; }
+    void SetHdr10Output(DXGI_HDR_METADATA_HDR10 hdr10) { m_hdr10Output = hdr10; }
```

Comment:
`SetHdr10Output` used only inside `DeviceResources` no need to declare one line method for single usage